factory.rb

```diff
@@ -1,139 +1,138 @@
-class Factory 
+class Factory
 
 	include Enumerable
```

Comment:
Remember about the styleguide:
- indentation: 2 whitespaces
- avoid tab indentation